src/javascript/JContent/ContentRoute/ContentBreadcrumb/ContentPath/ContentPath.jsx

```diff
@@ -20,10 +20,12 @@ const renderItems = (items, onItemClick) => {
 
 const ContentPath = ({items, onItemClick}) => {
     return (items.length > 0) &&
-        <Breadcrumb>
-            {renderItems(items, onItemClick)}
-            <></>
-        </Breadcrumb>;
+        <>
+            <Breadcrumb>
+                {renderItems(items, onItemClick)}
+            </Breadcrumb>
+            <Separator variant="vertical"/>
```

Comment:
Will this separator be properly styled as the ones inside the Moonstone's Breadcrumb without any additional changes?

Reply:
yes, as recommended by the UX